src/Instrumentation/Slim/src/SlimInstrumentation.php

```diff
@@ -29,7 +29,7 @@ class SlimInstrumentation
 
     public static function register(): void
     {
-        $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.slim');
+        $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.slim', schemaUrl: TraceAttributes::SCHEMA_URL);
```

Comment:
Should this be explicitly set as the schema that it was created against?

It may potentially be invalid against future versions of the spec 🤔

Reply:
You're right, I hadn't considered this. Since this package (opentelemetry-auto-slim), pulls in open-telemetry/sem-conv, which is the source of TraceAttributes, is it fair to assume that this package is compatible with whatever version of the schema is defined in open-telemetry/sem-conv?

Reply:
Apologies for my delayed response to this @sudol 

I have had a look through the documentation, and I think this may be the relevant part that was somewhere in the back of my mind: [Schema-File Driven Telemetry Producers](https://opentelemetry.io/docs/specs/otel/telemetry-stability/#schema-file-driven-telemetry-producers)

I think that we would be at risk of changing the schema version URL in a potentially unsafe way if `TraceAttributes::SCHEMA_URL` is used as is.

@brettmc did mention in the SIG this week that maybe we could provide a series of constants for these versions instead (tagged for his input/thoughts on this!)

Reply:
how does this look? https://github.com/open-telemetry/opentelemetry-php/pull/1301

Reply:
Nice @brettmc! I added a comment on there for discussion

Reply:
My interpretation of the spec re: schema_url is that we should explicitly set it to the latest schema version (`1.25.0`), confirm that we're setting the values correctly per that version, then leave it alone.

Reply:
@sudol - use `https://opentelemetry.io/schemas/1.25.0`, and the version should not change again after that.

Reply:
Ah yes. Sorry, I misunderstood. Yes I will get this updated tomorrow.